sdk/cosmosdb/cosmos/package.json

```diff
@@ -70,14 +70,14 @@
     "tsdocFlavor": "AEDoc"
   },
   "dependencies": {
+    "@azure/abort-controller": "1.0.0-preview.2",
```

Comment:
Tried adding the dependency with `rush add -p "@azure/abort-controller" --caret` however it complains about mismatched dependencies and  `--make-consistent` flag updates other packages. Is it okay use the `--exact` flag in this case? Do you have any suggestions? @HarshaNalluru , @ramya-rao-a

Reply:
Please pick the version that is being used in the other packages like eventhubs or storage which is "1.0.0-preview.2".
https://github.com/Azure/azure-sdk-for-js/blob/c28a1a51e6144655dc0d80242ba9b67d563a57c4/sdk/eventhub/event-hubs/package.json#L65

For the time being, `--caret` flag is mainly for importing new libraries that are not in our repo.